src/Components/HelpSolver.tsx

```diff
@@ -0,0 +1,13 @@
+export const HelpSolver = () => {
```

Comment:
We can export different components from the same file, should we maybe keep all Helper components in Help.tsx?

Reply:
Good idea. I'll look at it tomorrow

Reply:
The HelpSolver.tsx file has been deleted, and the HelpSolver code is now in Help.tsx